pkgs/tools/networking/aria2/default.nix

```diff
@@ -13,10 +12,10 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [ openssl c-ares libxml2 sqlite zlib libssh2 ] ++
-    stdenv.lib.optional stdenv.isDarwin Security;
+  buildInputs = [ openssl c-ares libxml2 sqlite zlib libssh2 ];
 
-  configureFlags = [ "--with-ca-bundle=/etc/ssl/certs/ca-certificates.crt" ];
+  configureFlags = [ "--with-ca-bundle=${cacert}/etc/ssl/certs/ca-bundle.crt" ] ++
+    stdenv.lib.optional stdenv.isDarwin [ "--with-openssl" "--without-appletls" ];
```

Comment:
I think I did try putting all flags aria2 put in their `makerelease-osx.mk`, but probably didn't try the `--with-ca-bundle` and `--with-apple-tls` combination. Will try again.